.env.test

```diff
@@ -0,0 +1,8 @@
+FIREBASE_ENV=TEST
+FIREBASE_API_KEY=AIzaSyCSaYXMSITnYcjWv_1i3Zn4BjZ9CiBqe_A
+FIREBASE_AUTH_DOMAIN=findadoc-test.firebaseapp.com
```

Comment:
Can a Map API key also be added? Not necessarily in this PR.

Reply:
definitely! Let's talk through .env files and how we want to manage them after this.